docs/monitoring.md

```diff
@@ -79,3 +79,21 @@ scrape_configs:
     action: replace
     target_label: kubernetes_name
 ```
+
+# Volume health monitoring
```

Comment:
You could have the doc like
```
# Monitoring volume health
DirectPV monitors volume health and reports abnormalities as event to PVCs and Pods. Currently volume mounts are monitored and more checks will be added later. This feature uses [Volume health monitoring CSI feature](https://kubernetes.io/docs/concepts/storage/volume-health-monitoring/).

To enable volume health monitoring, pass `--enable-volume-health-monitoring` flag to installation and `CSIVolumeHealth` feature gate must be enabled.
```

Reply:
I have also added the following because this image is required only when volume health monitor is enabled.

```
For private registries, please note that the following image is required for enabling volume health monitoring

quay.io/minio/csi-external-health-monitor-controller:v0.10.0

```